extension.cpp

```diff
@@ -907,7 +907,7 @@ struct callback_t final : prop_reference_t
 
 	void change_edict_state() noexcept
 	{
-		if(ref != INVALID_EHANDLE_INDEX) {
+		if(ref != (int)INVALID_EHANDLE_INDEX) {

```

Comment:
can you revert this i am gonna fix it later

Reply:
done